src/components/query-viewer-chart/controller/processDataAndMetadata.ts

```diff
@@ -512,7 +513,7 @@ function aggregatePoints(chartSerie: QueryViewerChartSerie) {
   });
   const value = aggregate(
     chartSerie.Aggregation,
-    currentYValues,
+    currentYValues.map(val => new GxBigNumber(val)),
```

Comment:
`[aggregatePoints]` It would be nice to add some tests using this utility with different values to validates the new behavior.